pyopenms_viz/_matplotlib/core.py

```
@@ -629,130 +595,163 @@ class MATPLOTLIBPeakMapPlot(MATPLOTLIB_MSPlot, PeakMapPlot):
             )
-            vlinePlot.generate(
-                z=z,
-                xlabel=self.xlabel,
-                ylabel=self.ylabel,
-                zlabel=self.zlabel,
-                **other_kwargs,
+            self.ax = vlinePlot.generate(None, None)
+
+        return self.ax
+
+    def create_main_plot(self):
```

Ruff (0.8.2)

Redefinition of unused `create_main_plot` from line 683

(F811)